build.sh

```diff
@@ -1,30 +1,32 @@
-#!/bin/sh
+#!/usr/bin/env bash
+
 set -o errexit
 
 # compile specified module
-modules=(collector)
+modules=(@collector)
```

Comment:
why add "@" here? Can't pass the execution

Reply:
Misunderstanding is to pass an array here.
It‘s wrong to use “@” here, ignore it plz.